### PR TITLE
Try built-in conda

### LINKED ISSUE #10: Built-in Conda or Docker ROOT job
The conda job that tests ROOT sometimes times out (6 hours!) instead of finishing in a few minutes. Not sure why yet. We can replace it with a docker powered job if it continues to fail.

_Originally posted by @henryiii in https://github.com/scikit-hep/vector/pull/9#issuecomment-605653481_

### PR DESCRIPTION
**Pull request by henryiii**
Closes #10 when it works, follow up to #9.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,20 +49,11 @@ jobs:
...
-          echo "notify_outdated_conda: false" > .condarc
-
-    - uses: goanpeca/setup-miniconda@v1
-      with:
-        miniconda-version: 4.7.12
-        environment-file: environment.yml
-        activate-environment: vector
-        condarc-file: .condarc
+    - name: Install environment
+      run: conda create env
```
Comment (chrisburr) [inline .github/workflows/ci.yml]:
```suggestion
      run: conda create env --file environment.yml --name vector
```

Comment (henryiii) [reply .github/workflows/ci.yml]: This should have been the default behavior? I'll try it...

Comment (henryiii) [reply .github/workflows/ci.yml]: My bad, I wrote this backwards! Sorry!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -49,20 +49,11 @@ jobs:
...
-          echo "notify_outdated_conda: false" > .condarc
-
-    - uses: goanpeca/setup-miniconda@v1
-      with:
-        miniconda-version: 4.7.12
-        environment-file: environment.yml
-        activate-environment: vector
-        condarc-file: .condarc
+    - name: Install environment
+      run: conda create env --file environment.yml --name vector
```
Comment (henryiii) [inline .github/workflows/ci.yml]:
```suggestion
      run: conda env create
```

**Pull request merged**